src/components/Organisms/CardMove/CardMove.scss

```diff
@@ -1,100 +1,95 @@
+@import "src/components/styles/global.scss";
 
-@import 'src/components/styles/global.scss';
-
-.scroller{
-    margin-top: 20px;
-    max-width: 1800px;   
+.scroller {
+  margin-top: 20px;
+  max-width: 1800px;
 }
-.scroller_inner{
-    display: flex;
-    gap: 30px;
-    padding-block: 1rem;
-    width: 1450px;
+.scroller_inner {
+  display: flex;
+  gap: 30px;
+  padding-block: 1rem;
+  width: 1450px;
 }
-.scroller[data-animated= "true"] {
-    overflow: hidden;
-   -webkit-mask: linear-gradient(90deg ,transparent, white 10%, white 90%, transparent);
+.scroller[data-animated="true"] {
+  overflow: hidden;
+  -webkit-mask: linear-gradient(
+    90deg,
+    transparent,
+    white 10%,
+    white 90%,
+    transparent
+  );
 }
-.scroller[data-animated= "true"] .scroller_inner{
-    
-    flex-wrap: nowrap;
-    animation: scroll 60s linear infinite;
+.scroller[data-animated="true"] .scroller_inner {
+  flex-wrap: nowrap;
+  animation: scroll 60s linear infinite;
 }
 
 @keyframes scroll {
-    to{
-        transform: translate(-50%);
-    }
+  to {
+    transform: translate(-50%);
+  }
 }
 
-
-.card{
-    background-color: $card-color;
-    border-radius: 18px;
-    height: 275px; 
+.card {
+  background-color: $card-color;
+  border-radius: 18px;
+  height: 245px;
 }
-.imgContainer{
-    width: 200px;
-    height: 150px;
+.imgContainer {
+  width: 200px;
+  height: 150px;
 }
-.imgCharacter{
-    display: block;  
-    width: 100%;
-    height: 100%;
+.imgCharacter {
+  display: block;
+  width: 100%;
+  height: 100%;
 }
-.nameCharacter{
-    margin-top: 10px;
-    color: var(--name-character);
-    font-family: "Poor Story", system-ui;
-    font-weight: 100;
-    font-size: 2.1rem;
-    text-align: center;
+.nameCharacter {
+  margin-top: 10px;
+  color: $color-black;
+  font-family: $font-PrincipalHeader;
+  font-weight: 100;
+  font-size: 1.5rem;
+  text-align: center;
 }
 
-.power{
-    padding: 0 10px;
-    margin: 10px 0 0 10px;
-    display: flex;
-    align-items: center;
-    gap: 5px;
-    height: 35px;
-    border-radius: 40px;
-    width:100px;
-    
+.power {
+  padding: 0 10px;
+  margin: -10px 0 0 10px;
+  display: flex;
+  align-items: center;
+  gap: 5px;
+  height: 35px;
+  border-radius: 40px;
+  width: 100px;
 }
-.containerButton{
-    display: flex;
-    align-items: center;
-    gap: 5px;
+.containerButton {
+  display: flex;
+  align-items: center;
+  gap: 5px;
 
-    & img{
-        width: 100%;
-        height: 100%;
-    }
+  & img {
+    width: 100%;
+    height: 100%;
+  }
 }
-.typeButton{
-    font-family: "Poppins", sans-serif;
+.typeButton {
+  font-family: $font-tittle;
```

Comment:
Typo, title*.
There's an extension in VS Code called Code Spell Checker, it displays an error when a word is misspelled, you should install i.